components/Tabs/Tabs.js

```diff
@@ -2,23 +2,26 @@
 class TabItem {
   constructor(element) {
     // attach dom element to object. Example in Tabs class
+    this.element = element;
   }
 
   select() {
     // should use classList
+    this.element.classList.add('Tabs__item-selected');
   }
 
   deselect() {
     // should use classList
+    this.element.classList.remove('Tabs__item-selected');
   }
 }
 
 class TabLink {
   constructor(element, parent) {
-    this.element;// attach dom element to object
-    this.tabs;// attach parent to object
-    this.tabItem;// assign this to the associated tab using the parent's "getTab" method by passing it the correct data
-    // reassign this.tabItem to be a new instance of TabItem, passing it this.tabItem
+    this.element = element;// attach dom element to object
```

Comment:
I suggest to be consistent with comment styles, some are on the side while others are above.